core/src/main/python/create.py

```diff
@@ -90,6 +90,7 @@ def __process_args(args):
     argument_map = cla_util.process_args(args, TOOL_TYPE_CREATE)
     __process_java_home_arg(argument_map)
     __process_domain_location_args(argument_map)
+    cla_helper.load_properties_file(argument_map)
```

Comment:
Are there any check for the validity of the values in the properties? What if entries are missing after user modifications?

Reply:
If the property file is missing, or a property is missing, then an internal timeout default is used.

Reply:
What happens if I specify  undeploy.timeout=180seconds?

Reply:
done